README.md

```diff
@@ -1,160 +1,90 @@
-[jStat](http://www.jstat.org/) - JavaScript Statistical Library
+**jStat** - JavaScript Statistical Library
 ===============================================================
 
-[![npm version](https://badge.fury.io/js/jStat.svg)](https://badge.fury.io/js/jStat)
-
-jStat provides native javascript implementations of statistical functions.
-Full details are available in the [docs](https://jstat.github.io/all.html).
+**jStat** provides native javascript implementations of statistical functions.
+Full details are available in the [docs](http://handsontable.github.io/jstat/).
 jStat provides more functions than most libraries, including the weibull, cauchy, poisson, hypergeometric, and beta distributions.
 For most distributions, jStat provides the pdf, cdf, inverse, mean, mode, variance, and a sample function, allowing for more complex calculations.
 
-Using jStat in a Browser
-------------------------
+[![npm version](https://img.shields.io/npm/v/@handsontable/jstat.svg)](https://www.npmjs.com/package/@handsontable/jstat)
 
-jStat can be used in the browser. The `jStat` object will be added to the window. For example:
+<br/>
 
-```
-<script src="components/jstat.js"></script> <!-- include jStat, from the CDN or otherwise -->
+## Table of contents
 
-<script>
-...
-var jstat = this.jStat(dataset); // jStat will be added to the window
-...
-data[i]['cum'] = jstat.normal(jstat.mean(), jstat.stdev()).cdf(data[i].x);
-...
-</script>
 
-```
+ 1. [Installation](#installation)
+ 2. [Getting Started](#getting-started)
+ 3. [Building jStat](#building-jstat)
+ 4. [Running Tests](#Running-tests)
+ 5. [Get the Code](#get-the-code)
+ 6. [Contributing](#contributing)
 
-CDN
----
+<br/>
```

Comment:
Either we add spacing between sections or not. We have a mix now. I would remove `<br>` as we did in our main repo.